docs/source/github_workflow.rst

```diff
@@ -0,0 +1,37 @@
+Release Workflow
+==================
+
+When preparing a new release, developers need to be aware of the existence of multiple processes.
```

Comment:
can you describe the process ?

Reply:
I will, this is just a test to check if things work. I will write the documentation when I am sure that all is right.